src/Paysera/WalletApi/Mapper.php

```diff
@@ -1482,10 +1482,49 @@ public function decodeUser($data)
         if (isset($data['locale'])) {
             $this->setProperty($user, 'locale', $data['locale']);
         }
+        if (isset($data['pep'])) {
+            $result = array();
```

Comment:
Why not use short syntax?

Reply:
maybe $politicallyExposedPersons or $peps instead of $result?

Reply:
```
    "require": {
        "php": ">=5.5 || >=7.0",
```